CadRevealComposer/Operations/Tessellating/TessellationUtils.cs

```diff
@@ -1,13 +1,13 @@
 ﻿namespace CadRevealComposer.Operations.Tessellating;
 
-using CadRevealComposer.Primitives;
-using CadRevealComposer.Tessellation;
-using CadRevealComposer.Utils;
+using Primitives;
 using System;
 using System.Collections.Generic;
 using System.Drawing;
 using System.Linq;
 using System.Numerics;
+using Tessellation;
+using Utils;
```

Comment:
Pretty sure the import sorting order is wrong? 🤔

Reply:
Consider updating csharpier to the latest version and running that on all files.